targets/tofino/control_plane_architecture.h

```diff
@@ -54,13 +54,14 @@ struct ActionProfileTraits<Arch::TNA> {
     static const cstring name() { return "action profile"; }
     static const cstring propertyName() { return "tna_implementation"; }
     static const cstring typeName() { return "ActionProfile"; }
-    static const cstring sizeParamName() { return "size"; }
+    static const cstring sizeParamName() { return "size"; }  // FIXME: this is not correct
```

Comment:
Do you know what the param is here?

Reply:
Ah, I forgot to remove the comment. I later realized this is actually correct.